regression/tests/helpers/api_clients.py

```diff
@@ -435,6 +436,28 @@ def get_coupon_codes(self, coupon_id):
         return coupon_codes
 
 
+class EnrollmentApiClient(EdxRestApiBaseClass):
+    """
+    Enrollment API client
+    """
+
+    api_url_root = ENROLLMENT_API_URL
+    append_slash = False
+
+    def is_user_enrolled(self, username, course_run_id):
+        """ Get enrollment details for the given user and course run.
+        Args:
```

Comment:
nit: Please update docstrings format: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
This is a large repo and there would be lots of functions using that format, I will create a seperate pr just for this issue

Reply:
Well we should start the right thing when ever we can :)